main.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'app'
+def display_menu
+  puts 'Please choose an option by entering a number from 1 to 7:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def handle_menu_option(option, app)
+  case option
+  when 1
+    app.list_books
+  when 2
+    app.list_people
+  when 3
+    app.create_person
+  when 4
```

Comment:
- [ ] Dear @Divinity-dev, Good job adding the classes. But at this point, I could not proceed to create a person no matter what option I provide for the `Has parent permission? [Y/N]:` section. Please make sure your app is successful in adding a person and preceding to other functionalities.
- [ ] Because of this, I could not proceed to check other functionality of your app. Hence when you submit for a review all the functionalities will be reviewed against the provided requirements. It would be great if people manage to complete the adding person. alongside rental functionality for the books. 
- [ ] Also, one more thing is that your app asks for a `Classroom:` which is not a requirement for this milestone. Also, remember when you initialize the app, we don't have any class initialized, hence there is no possible way we can assign the student to a non-existing class.
- [Optional] Please try to separate your code into different levels of commits. Remember your commits are going to help you look back and debug your code. It's a good idea to commit after every visible change.
  
![image](https://user-images.githubusercontent.com/69077061/233657646-eabb6415-3afc-414c-9e40-4a6cb56616af.png)